pkg/skaffold/build/local/types.go

```diff
@@ -32,15 +32,14 @@ import (
 
 // Builder uses the host docker daemon to build and tag the image.
 type Builder struct {
-	cfg *latest.LocalBuild
-
+	cfg          *latest.LocalBuild
 	localDocker  docker.LocalDaemon
 	localCluster bool
 	pushImages   bool
-	prune        bool
-	skipTests    bool
-	kubeContext  string
-	builtImages  []string
+
+	builtImages []string
+
+	runCtx *runcontext.RunContext
```

Comment:
I'm a little worried about this being a pointer, since we really shouldn't ever be modifying anything in this context after it's created by the runner. can this just be `runCtx runcontext.RunContext` instead?

Reply:
That is a fair point, I'll rework that. 
If we are serious about that - wouldn't we have to change `runContext` having pointers itself as well?
Also what is the overhead of passing by value vs pointer all around the place I wonder...